server/block/stone_brick_stairs.go

```diff
@@ -0,0 +1,76 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/block/model"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/go-gl/mathgl/mgl64"
+)
+
+// StoneBrickStairs are blocks that allow entities to walk up blocks without jumping. They are crafted using end bricks.
+type StoneBrickStairs struct {
```

Comment:
Stone brick stairs can also be `mossy`. Consider adding either a `Mossy` bool, or, similar to `StoneBricks`, a `Type StoneBricksType` field. The identifier for mossy stone bricks is `minecraft:mossy_stone_brick_stairs`.

Reply:
I've added the mossy variant using a boolean.